integrations/oci/tls-certificates/src/test/resources/logging.properties

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+.level = INFO
+
+handlers = java.util.logging.ConsoleHandler
+
+io.helidon.integrations.oci.sdk.cdi.level = FINE
```

Comment:
Do not add FINE logging to any `logging.properties` in the repo.
Also this is strange ordering - please follow other files in the repo, to be aligned with what everybody else uses